backend/graph/api/mutations.resolvers.go

```diff
@@ -165,6 +165,40 @@ func (r *mutationRootResolver) SetEpisodeProgress(ctx context.Context, id string
 	return e, err
 }
 
+// SetShortProgress is the resolver for the setShortProgress field.
+func (r *mutationRootResolver) SetShortProgress(ctx context.Context, id string, progress *float64, duration *float64) (*model.Short, error) {
+	p, err := getProfile(ctx)
+	if err != nil {
+		return nil, err
+	}
+	s, err := r.QueryRoot().Short(ctx, id)
+	if err != nil {
+		return nil, err
+	}
+	if progress != nil {
+		d := 0.0
+		if duration != nil {
+			d = *duration
+		}
+		err = r.GetQueries().SaveVideoProgress(ctx, sqlc.SaveVideoProgressParams{
+			ProfileID: p.ID,
+			Progress:  float32(*progress),
```

Comment:
Does it make sense to have a 0 progress?
IMO 0 == removal OR "NOP"

Reply:
nil is removal, 0 could be that you've clicked on the video. 
not a bad thing to track